### PR TITLE
693 fix plant names

### PR DESCRIPTION
**Pull request by temmey**
## Basics

- [x] I added a line to [/doc/CHANGELOG.md](/doc/CHANGELOG.md)
- [x] The PR is rebased with current master.
- [x] Details of what you changed are in commit messages.
- [x] References to issues, e.g. `close #X`, are in the commit messages.
- [ ] The buildserver is happy.

## Checklist

- [x] I have installed and I am using [pre-commit hooks](../doc/contrib/README.md#Hooks)
- [ ] I fully described what my PR does in the documentation
      (not in the PR description)
- [ ] I fixed all affected documentation
- [ ] I fixed all affected decisions
- [ ] I added unit tests for my code
- [x] I added code comments, logging, and assertions as appropriate
- [ ] I translated all strings visible to the user
- [x] I mentioned [every code or binary](https://github.com/ElektraInitiative/PermaplanT/blob/master/.reuse/dep5) not directly written or done by me in [reuse syntax](https://reuse.software/)
- [ ] I created left-over issues for things that are still to be done
- [x] Code is conforming to [our Architecture](/doc/architecture)
- [x] Code is conforming to [our Guidelines](/doc/guidelines)
      (exceptions are documented)
- [x] Code is consistent to [our Design Decisions](/doc/decisions)

## Review

- [ ] I've tested the code
- [ ] I've read through the whole code
- [ ] Examples are well chosen and understandable

### CODE REVIEW

**Comment by temmey** [comment]
jenkins build please

**Comment by markus2330** [comment]
jenkins build please

File: frontend/src/features/map_planning/components/Map.tsx [hunk 1]
```diff
@@ -107,6 +111,16 @@ export const Map = ({ layers }: MapProps) => {
               >
                 <GridIcon></GridIcon>
               </IconButton>
+              <IconButton
```
Comment (markus2330) [inline frontend/src/features/map_planning/components/Map.tsx:114]: Please comment out/revert so that we can merge this.

**Review by markus2330 — CHANGES_REQUESTED** [review]
Great work. Only the eye should either be functional or not be there.

**Comment by markus2330** [comment]
Great job, good to have this!

**Pull request merged**